product-service/src/functions/createProduct/index.ts

```diff
@@ -1,19 +1,26 @@
-import schema from './schema';
 import { handlerPath } from '@libs/handler-resolver';
+import schema from './schema';
+
+
 
 export default {
+
   handler: `${handlerPath(__dirname)}/handler.main`,
   events: [
     {
       http: {
         method: 'post',
-        path: 'hello',
+        path: 'products',
         request: {
```

Comment:
nit: you need an additional property to generate a request body in swagger:
"bodyType: 'CreateProductRequestBody' where CreateProductRequestBody defined as interface/type